### ISSUE TITLE
Settings: No way to focus edit button via keyboard

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52469]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : HighSierra10.13.5

**Additional Details:**
MAS Violated : MAS 2.1.1

**Repro Steps:**
1.Launch VS Code.
2.Navigate to Settings icon in Activity Bar->Press Enter->Select "Settings" from dropdown.
3.Keyboard Focus does not go on the "Edit" button present when a particular list item is expanded on the screen.

**Actual:**
Keyboard Focus does not go on the "Edit" button present when a particular list item is expanded and the edit button is visible on the page.

**Expected:**
Keyboard Focus does not go on the "Edit" button present when a particular list item is expanded and the edit button is visible on the page.

When the edit button is expanded using the mouse click , two options are displayed(True/False) which cannot be used until the edit button is actionable.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The End user will not be able to access the edit buttons which is embedded with some functionalities.

**MAS Reference:**
MAS 2.1.1 - https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

**Attachment for Reference:**
[A11y_VSCode_UserSettings_Keyboard_FocusIssue.pptx](https://github.com/Microsoft/vscode/files/2119268/A11y_VSCode_UserSettings_Keyboard_FocusIssue.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This is the settings editor, leaving to @sandy081 to decide how to tackle and if low hanging

**Comment by sandy081** [comment]
This settings editor is based on text editor. Since we are coming up with a new settings editor which is web style and will be accessible, I would not invest here.

**Comment by isidorn** [comment]
That makes sense, assigning to backlog.
